qml/InitDSMonitorDialog.qml

```diff
@@ -0,0 +1,425 @@
+            }
+            Label {
+                id: discoveryServerTransportProtocolLabel
+                text: "Transport Protocol"
+                Layout.preferredWidth: parent.width / 3
+                InfoToolTip {
+                    text: 'Transport protocol of the\n' +
+                          'Discovery Server network\n' +
+                          'address.'
+                }
+            }
+            Label {
+                id: discoveryServerIPLabel
+                text: "IP"
+                Layout.preferredWidth: parent.width / 3
+                InfoToolTip {
+                    text: 'IP of the Discovery Server\n' +
+                          'network address.\n'
+                }
+            }
+            Label {
+                id: discoveryServerPortLabel
+                text: "Port"
+                Layout.preferredWidth: parent.width / 5
+                InfoToolTip {
+                    text: 'Port of the Discovery Server\n' +
+                          'network address.'
+                }
+            }
+            Rectangle {
+                id: discoveryServerRemoveIcon
+                Layout.fillWidth: true
+                color: "Blue"
+            }
+            Rectangle {
+                Layout.preferredWidth: parent.width
+                Layout.columnSpan: parent.columns
+                height: 2
+                color: Theme.grey
+            }
+
+            Rectangle {
+                id: dsLocatorsRect
+                Layout.columnSpan: 4
+                Layout.fillWidth: true
+                height: 100
+                color: "transparent"
+
+                ListView {
+                    id: discoveryServerLocatorsListView
+                    model: discoveryServerLocatorsModel
+                    delegate: discoveryServerLocatorsDelegate
+                    anchors.fill: parent
+                    clip: true
+                    spacing: 5
+
+                    ScrollBar.vertical: ScrollBar {
+                        id: scrollBar
+                        visible: true
+                        hoverEnabled: true
+                        policy: Qt.ScrollBarAlwaysOn
+                    }
+
+                    onCountChanged: {
+                        var newIndex = count - 1
+                        positionViewAtEnd()
+                        currentIndex = newIndex
+                    }
+                }
+
+                Component {
+                    id: discoveryServerLocatorsDelegate
+
+                    Item {
+                        width: discoveryServerLocatorsListView.width
+                        height: dsLocatorRow.height
+
+                        RowLayout {
+                            id: dsLocatorRow
+                            spacing: 10
+
+                            ComboBox {
+                                id: discoveryServerTransportProtocol
+                                model: ["UDPv4", "UDPv6", "TCPv4", "TCPv6"]
```

Comment:
Get this info from Backend (as unit magnitudes).

Reply:
Addressed in a4e57e8